### PR TITLE
docs: remove duplicate message in aws billing integration page

### PR DESCRIPTION
**Pull request by OrekiYuta**
## What does this PR do?

Removed duplicate message in aws billing integration description page
https://www.elastic.co/docs/current/integrations/aws/billing

## Checklist

- [x]  I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x]  I have verified that all data streams collect metrics or logs.
- [x]  I have added an entry to my package's changelog.yml file.
- [x]  I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## How to test this PR locally

- Clone integrations repo.
- Install elastic-package locally.
- Start elastic stack using elastic-package.
- Move to integrations/packages/aws directory.
- Run the following command to run tests. elastic-package test

## Screenshots
https://www.elastic.co/docs/current/integrations/aws/billing#metrics-reference
image

### CODE REVIEW

File: packages/aws/_dev/build/docs/billing.md [hunk 1]
```diff
@@ -47,8 +47,6 @@ For step-by-step instructions on how to set up an integration, see the
 
 The `billing` data stream collects billing metrics from AWS.
 
-An example event for `billing` looks as following:
```
Comment (aliabbas-elastic) [inline packages/aws/_dev/build/docs/billing.md:50]: You also need to run `elastic-package build` after doing this change

File: packages/aws/changelog.yml [hunk 2]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "2.16.1"
```
Comment (aliabbas-elastic) [inline packages/aws/changelog.yml:2]: Please update the same version in manifest.yml

File: packages/aws/changelog.yml [hunk 3]
```diff
@@ -1,4 +1,9 @@
...
+  changes:
+    - description: Fix AWS Billing Metrics reference description 
+      type: enhancement
```
Comment (aliabbas-elastic) [inline packages/aws/changelog.yml:5]: Lets treat it as a bugfix and not an enhancement

File: packages/aws/changelog.yml [hunk 4]
```diff
@@ -1,4 +1,9 @@
...
+      link: https://github.com/elastic/integrations/pull/10382/
```
Comment (aliabbas-elastic) [inline packages/aws/changelog.yml:6]:
```suggestion
      link: https://github.com/elastic/integrations/pull/10382
```

**Review by aliabbas-elastic — CHANGES_REQUESTED** [review]

File: packages/aws/_dev/build/docs/billing.md (hunk 1 above)
Comment (OrekiYuta) [reply packages/aws/_dev/build/docs/billing.md:50]: already run the commond testing in local

File: packages/aws/changelog.yml (hunk 2 above)
Comment (OrekiYuta) [reply packages/aws/changelog.yml]: updated

File: packages/aws/changelog.yml (hunk 3 above)
Comment (OrekiYuta) [reply packages/aws/changelog.yml]: updated

**Review by OrekiYuta — COMMENTED** [review]
requested change

File: packages/aws/changelog.yml [hunk 5]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "2.17.1"
+  changes:
+    - description: Fix AWS Billing Metrics reference description
```
Comment (aliabbas-elastic) [inline packages/aws/changelog.yml:4]:
```suggestion
    - description: Fix AWS Billing Metrics reference description.
```

**Review by aliabbas-elastic — APPROVED** [review]
LGTM

**Pull request closed without merging**